src/transformers/models/whisper/tokenization_whisper.py

```diff
@@ -605,21 +638,19 @@ def decode(
             )
         # retrieve offsets
         if output_offsets:
-            offsets = None
             offsets = self._compute_offsets(token_ids, time_precision=time_precision)
```

Comment:
We want to use `token_ids` here, not `filtered_ids`, since we need the timestamp ids to be present so that we can compute the offsets